docs/guides/setup-channel.mdx

```diff
@@ -49,55 +48,47 @@ Available plans:
       The Instances section in your [IBM Quantum account page](https://quantum.ibm.com/account) lists the instances that you can access.
     </Admonition>
 
-1. Retrieve your IBM Quantum token from the [IBM Quantum account page](https://quantum.ibm.com/account).
-
-1. <span id="save-account"></span>Authenticate to the service by calling `QiskitRuntimeService` with your IBM Quantum API key and CRN:
-
-   ```python
-   from qiskit_ibm_runtime import QiskitRuntimeService
-
-   service = QiskitRuntimeService(channel="ibm_quantum", token="<MY_IBM_QUANTUM_TOKEN>")
+1. Retrieve your API token from the [IBM Quantum account page](https://quantum.ibm.com/account), and activate your Python virtual environment.  See the [installation instructions](/guides/install-qiskit#local) if you do not already have a virtual environment set up. 
 
-   ```
-
-   Or, optionally use the `save_account()` method to save your credentials for easy access later on, before initializing the service.
+    <span id="save-account"></span>**If you are working in a trusted Python environment (such as on a personal laptop or workstation),** use the `save_account()` method to save your credentials locally. ([Skip to the next step](#untrusted-machine) if you are not using a trusted environment, such as a shared or public computer, to authenticate to IBM Quantum Platform.) To use `save_account()`, run Python in your virtual environment to open a read-eval-print loop (REPL), then enter the following:
 
     ```python
     from qiskit_ibm_runtime import QiskitRuntimeService
 
-    # Save an IBM Quantum account and set it as your default account.
     QiskitRuntimeService.save_account(
-        channel="ibm_quantum",
-        token="<MY_IBM_QUANTUM_TOKEN>",
-        set_as_default=True,
-        # Use `overwrite=True` if you're updating your token.
-        overwrite=True,
+      token="<your-token">,
+      channel="ibm_quantum", # `channel` distinguishes between different account types
+      overwrite=True # Use `overwrite=True` if you're updating your token
     )
```

Comment:
If we are recommending people to use REPL, then the code snippet should be structured in a way that can be copy&pasted into the REPL and the token modified easily. So I'd recommend changing this to 3 blocks of code:

block 1:
```python 
from qiskit_ibm_runtime import QiskitRuntimeService
```

block 2:
```python
token = "<your-token>"
```

block 3:
```python
QiskitRuntimeService.save_account(token=token, channel="ibm_quantum")
```

Personally I don't like `overwrite=True`. The current code block has `token="<your-token">`, which presumably meant to be `token="<your-token>",`. If this was fixed, and I copy&paste the block into my REPL, it'd overwrite my saved account with `<your-token>` as the token!

Reply:
removed the overwrite and split into two blocks f20da5afcad797f65a9b408462abdde613390f0a and 84de35b3570afb1dee53952ed6c74136a7603a79